tests/unittest_qmail-send.c

```diff
@@ -0,0 +1,239 @@
+#define DEPRECATED_FUNCTIONS_REMOVED
+#include <check.h>
+#include "qmail-send-without-main.c"
+
+extern struct job *jo;
+extern int numjobs;
+
+// START test helpers for the API we might prefer from qmail-send
+
+#define available 0
+#define unavailable 1
+
+static void job_init_this_many(int numslots) {
```

Comment:
Please put the ```{``` for functions on a line of it's own as the other code also does.